cmd/uninstall.go

```diff
@@ -41,5 +43,6 @@ var UninstallCmd = &cobra.Command{
 
 func init() {
 	UninstallCmd.Flags().BoolVar(&uninstallKubernetes, "kubernetes", false, "Uninstall Dapr from a Kubernetes cluster")
+	UninstallCmd.Flags().BoolVar(&uninstallAll, "all", false, "Remove the redis container as well")
```

Comment:
This is tied to redis container. There could be other containers as well that we like to uninstall.

Reply:
I merged the PR before this comment, but I think it's fine to wait to update that later if there are others.